lib/routes/account.js

```diff
@@ -950,26 +955,31 @@ module.exports = function (
             return getGeoData(ip)
               .then(
                 function (geoData) {
-                  return mailer.sendVerifyLoginEmail(
-                    emailRecord,
-                    tokenVerificationId,
-                    {
-                      acceptLanguage: request.app.acceptLanguage,
-                      flowId: flowId,
-                      flowBeginTime: flowBeginTime,
-                      ip: ip,
-                      location: geoData.location,
-                      redirectTo: redirectTo,
-                      resume: resume,
-                      service: service,
-                      timeZone: geoData.timeZone,
-                      uaBrowser: sessionToken.uaBrowser,
-                      uaBrowserVersion: sessionToken.uaBrowserVersion,
-                      uaOS: sessionToken.uaOS,
-                      uaOSVersion: sessionToken.uaOSVersion,
-                      uaDeviceType: sessionToken.uaDeviceType
-                    }
-                  )
+                  return db.accountEmails(sessionToken.uid)
+                    .then((emails) => {
+                      sessionToken.emails = emails
+                      return mailer.sendVerifyLoginEmail(
```

Comment:
Careful, we don't want to send to *unverified* emails addresses here, or we're make a security hole.